### DISCUSSION TITLE
optimum number of bootnodes?

### DISCUSSION THREAD

**Discussion by ghost** [0xPolygon/polygon-edge#947] [Q&A]
Good day, Can anyone tell me what the optimum number of bootnodes is, and why?

Thanks

**Comment by laviniat1996** [comment]
Hi @Nexsuperne101 ,

It depens on the number of validators. The minimum number of bootnodes is 1, but more bootnodes provide  resilience to the network in case of outages. For example, for a 4 validators network, 2 bootnodes would be a good number.
